spec/models/room_spec.rb

```diff
@@ -3,20 +3,32 @@
 RSpec.describe Room, type: :model do
   context "basic tests" do
     let(:room) { FactoryGirl.build(:room) }
+    let(:room2) { FactoryGirl.build_stubbed(:room) }
+
     it "has a valid constructor" do
       expect(room).to be_valid
     end
 
-    it "is not valid without a name" do
+    it "is invalid without a name" do
       room.name = ""
       expect(room).not_to be_valid
       expect(room.errors[:name]).to include "can't be blank"
     end
 
+    it "is invalid with a duplicate name" do
+      room.name  = "Nuestra señora del Pilar"
+      room2.name = "Nuestra señora del Pilar"
+      room2.valid?
+      expect(room2.errors[:name]).not_to include "can't have duplicate name"
+    end
+
     it "has a positive capacity" do
       room.capacity = 0
       expect(room).not_to be_valid
       expect(room.errors[:capacity]).to include "must be greater than 0"
+    end
+
+    it "has not negative capacity" do
```

Comment:
Thanks for your review @sergio-ocon.
First, I must focus in the functionality of my PR, and then I can make a new PR for refactoring. I will keep your comments for the future.